src/main/groovy/org/pih/warehouse/product/ProductSupplierFilterCommand.groovy

```diff
@@ -23,6 +23,8 @@ class ProductSupplierFilterCommand extends PaginationCommand implements Validate
 
     String order
 
+    Boolean unlimitedMax = false
```

Comment:
maybe you could just override the getMax method from the `PaginationCommand` and that would be cleaner? I don't have hard opinion though, so I'll wait for guys' opinion too.